src/backend/feed.js

```diff
@@ -0,0 +1,33 @@
+const storage = require('./utils/storage');
+
+/**
+ * Feed Object, upon being created all feeds will be active and have a feedId of 0 until saved into Redis
+ * @param url url of feed
+ * @param name author name of feed
+ * @param feedId unique id for feed
+ */
+class Feed {
+  constructor(url, name) {
+    this.url = url;
+    this.name = name;
+    this.inactive = false;
+    this.feedId = 0;
+  }
+
+  /**
+   * Saves current feed to appropriate set in Redis
+   */
+  async save() {
+    this.feedId = await storage.addFeed(this);
```

Comment:
Let's move to using the feed url as the key.

Reply:
in storage.js did you want to change this to be the case as well?

Reply:
I filed #522 to do this, if you want to leave it til later.

Reply:
I'll leave this one for later